js_modules/dagster-ui/packages/ui-core/src/search/useGlobalSearch.tsx

```diff
@@ -355,13 +369,17 @@ export const useGlobalSearch = ({includeAssetFilters}: {includeAssetFilters: boo
     if (!secondaryData) {
       return;
     }
-    const results = secondaryDataToSearchResults({data: secondaryData}, includeAssetFilters);
+    const results = secondaryDataToSearchResults(
+      {data: secondaryData},
+      includeAssetFilters,
+      usersByEmail,
```

Comment:
Maybe make all of these arguments be in an object? thinking about [this stank](https://github.com/dagster-io/internal/discussions/9682)